example/config/spiffe-csi-driver.yaml

```diff
@@ -1,3 +1,117 @@
+---
+
```

Comment:
Do we need the leading `---`?  The other files don't have it.

Reply:
That's not necessary